cobigen/cobigen-core/src/main/java/com/devonfw/cobigen/impl/util/ConfigurationFinder.java

```diff
@@ -102,12 +102,22 @@ public static URI findTemplatesLocation() {
       // use new template set configuration
       Path templateSetsFolderLocation = getTemplatesFolderLocation(cobigenHome, configFile, templateSetsLocation);
       if (templateSetsFolderLocation != null && Files.exists(templateSetsFolderLocation)) {
+        if (!Files.exists(templateSetsFolderLocation)) {
+          LOG.warn(
+              "The last value of the template-set path, does not exist. Using home directory:{} to search for templates",
+              cobigenHome);
+        }
         return templateSetsFolderLocation.toUri();
       }
 
       // use old templates configuration
       Path templatesFolderLocation = getTemplatesFolderLocation(cobigenHome, configFile, templatesLocation);
-      if (templatesFolderLocation != null && Files.exists(templatesFolderLocation)) {
+      if (templatesFolderLocation != null) {
+        if (!Files.exists(templatesFolderLocation)) {
```

Comment:
why do you think is this an equivalent change?
Now, `templatesFolderLocation.toUri()` will always be called although the `templatesFolderLocation` might not exist.